src/gsAssembler/gsExprEvaluator.h

```diff
@@ -324,13 +325,19 @@ class gsExprEvaluator
                        std::string const & fn)
     { writeParaview_impl<E,false>(expr,m_exprdata->getMap(),fn); }
 
+    template<class E>
+    std::vector<std::string> expr2vtk(const expr::_expr<E> & expr, std::string label="SolutionField",unsigned nPts=1000, unsigned precision=5);
+
+    std::vector<std::string> geoMap2vtk(const geometryMap geoMap, unsigned nPts=1000, unsigned precision=5);
```

Comment:
This funtion geoMap2vtk is never used, is it needed ?

Reply:
Indeed since you suggested to store a `gsMultiPatch` as a member of `gsParaviewDataSet` and not a `gsGeometryMap` this function is not used anywhere, so I guess it should be removed altogether